components/blocks/FilesToSections.vue

```diff
@@ -38,11 +38,6 @@ export default {
     DTOC,
   },
   filters: {
-    humanize(str) {
-      const cleanStr = str.replace(/-/g, ' ');
-      const upperFirst = cleanStr.charAt(0).toUpperCase() + cleanStr.slice(1);
-      return upperFirst;
-    },
     urlize(str) {
```

Comment:
also get rid of the urlize filter (we shouldn't use filters anywhere), change to the method use if needed